Rakefile

```diff
@@ -1,6 +1,37 @@
 require 'bundler'
 require 'bundler/gem_tasks'
+require 'bump'
+require 'github_changelog_generator'
 
 task :spec do
   require File.expand_path('spec/run')
 end
+
+desc 'Sanity Checks and releases gem to rubygems'
+task :sanity do
```

Comment:
This block doesn't read like a "sanity check" block - it's actually making changes. Call it `:release` or something else that more clearly states what it's doing.